sim/src/tlv.rs

```diff
@@ -117,3 +117,41 @@ impl TlvGen {
         result
     }
 }
+
+/* Autogenerated by imgtool.py, do not edit. */
```

Comment:
Autogenerated code has no place in the same file as non-autogenerated code. This isn't a .emacs file ;)

Reply:
What do you propose? The intent is to show that this data is generated from the public key and pasted in? It's already inserted just like this in the keys.c file.

Reply:
I've moved the key into its own file, and used `include!("path")`.